apps/web/ui/partners/confirm-payouts-sheet.tsx

```
@@ -362,6 +422,37 @@ function ConfirmPayoutsSheetContent() {
+                ),
+              tooltipContent: (
+                <div className="max-w-xs px-4 py-2 text-center text-sm text-neutral-700">
+                  Payouts that are processed externally via the{" "}
+                  <code className="rounded-md bg-neutral-100 px-1 py-0.5 font-mono">
+                    payout.confirmed
+                  </code>{" "}
+                  webhook event.
+                  <a
+                    href="http://dub.co/docs/partners/external-payouts"
+                    target="_blank"
+                  >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)